### ISSUE TITLE
Getting this.getPlotly(...).react is not a function   error

### ISSUE THREAD

**Issue by Arjun-Shinojiya** [plotly/angular-plotly.js#74]
I am trying to integrate plotly via CDN,for that I imported PlotlyViaCDN module and version is **PlotlyViaCDNModule.plotlyVersion = '1.5.0';** . I am getting plotly graph but with graph I am getting this error too. I dont know how to resolve it . Please help. I am using Angular 8

**Comment by andrefarzat** [comment]
Hello, @Arjun-Shinojiya, I can't understand what is the issue. Could you show us the code snippet and the error message you're getting ? 😸

**Comment by Arjun-Shinojiya** [comment]
**this.getPlotly(...).react is not a function** @andrefarzat this is the message I am getting. The  error is in angular-plotly.js.js files 363 line .
```
 return this.plotly.update(this.plotlyInstance, this.data, layout, this.config, this.frames).then(() => {

            const figure = this.createFigure();
            this.update.emit(figure);
        }
```
Snippet is showing error here

**Comment by andrefarzat** [comment]
@Arjun-Shinojiya I still can't replicate this bug here. I created a project from scratch using angular 8.x and it worked fine following the [quick start guide](https://github.com/plotly/angular-plotly.js#quick-start) with `PlotlyViaCDNModule`.

Could you also show us your `app.module.ts` and `package.json` files?

**Comment by Arjun-Shinojiya** [comment]
Hi, @andrefarzat  thank you for reply. Here is my package.json code. please check it.
```
{
    "name": "dve-frontend",
    "version": "0.0.0",
    "scripts": {
        "ng": "ng",
        "start": "ng serve --aot",
        "build": "ng build --prod",
        "test": "ng test",
        "lint": "ng lint",
        "e2e": "ng e2e"
    },
    "private": true,
    "dependencies": {
        "5": "0.0.1",
        "@angular/animations": "^8.0.0",
        "@angular/cdk": "^8.0.1",
        "@angular/common": "~8.0.0",
        "@angular/compiler": "~8.0.0",
        "@angular/core": "~8.0.0",
        "@angular/flex-layout": "^8.0.0-beta.26",
        "@angular/forms": "~8.0.0",
        "@angular/material": "^8.0.1",
        "@angular/platform-browser": "~8.0.0",
        "@angular/platform-browser-dynamic": "~8.0.0",
        "@angular/router": "~8.0.0",
        "@ng-bootstrap/ng-bootstrap": "^4.2.1",
        "angular-datatables": "^8.0.0",
        "angular-gridster2": "^8.0.0",
        "angular-plotly.js": "^1.3.2",
        "angular2-toaster": "^7.0.0",
        "bootstrap": "^4.3.1",
        "bootstrap4": "0.0.1-security",
        "datatables.net": "^1.10.19",
        "datatables.net-dt": "^1.10.19",
        "font-awesome": "^4.7.0",
        "jquery": "^3.4.1",
        "lodash": "^4.17.15",
        "moment": "^2.24.0",
        "ng-connection-service": "^1.0.4",
        "ng-http-loader": "^5.0.0",
        "ng2-pdf-viewer": "^5.3.2",
        "ngx-bootstrap": "^4.2.0",
        "ngx-cookie-service": "^2.2.0",
        "ngx-showdown": "^5.0.0",
        "ngx-slick-carousel": "^0.4.4",
        "ngx-smart-modal": "^7.1.1",
        "ngx-toastr": "^10.0.4",
        "ngx-toggle-switch": "^2.0.5",
        "plotly.js": "^1.43.1",
        "popper.js": "^1.15.0",
        "rgb-hex": "^3.0.0",
        "rxjs": "~6.4.0",
        "showdown": "^1.9.0",
        "slick-carousel": "^1.8.1",
        "tslib": "^1.9.0",
        "zone.js": "~0.9.1"
    },
    "devDependencies": {
        "@angular-devkit/build-angular": "~0.800.0",
        "@angular/cli": "~8.0.0",
        "@angular/compiler-cli": "~8.0.0",
        "@angular/language-service": "~8.0.0",
        "@fortawesome/fontawesome-free": "^5.9.0",
        "@types/datatables.net": "^1.10.17",
        "@types/jasmine": "~3.3.8",
        "@types/jasminewd2": "~2.0.3",
        "@types/jquery": "^3.3.29",
        "@types/lodash": "^4.14.136",
        "@types/node": "~8.9.4",
        "@types/showdown": "^1.9.3",
        "codelyzer": "^5.0.0",
        "jasmine-core": "~3.4.0",
        "jasmine-spec-reporter": "~4.2.1",
        "karma": "~4.1.0",
        "karma-chrome-launcher": "~2.2.0",
        "karma-coverage-istanbul-reporter": "~2.0.1",
        "karma-jasmine": "~2.0.1",
        "karma-jasmine-html-reporter": "^1.4.0",
        "protractor": "~5.4.0",
        "ts-node": "~7.0.0",
        "tslint": "~5.15.0",
        "typescript": "~3.4.3"
    }
}
```

**Comment by Arjun-Shinojiya** [comment]
@andrefarzat  I tried PlotlyViaWindowModule and no error is showing with window module. Just issue is with plotlyviaCDN module.

**Comment by andrefarzat** [comment]
@Arjun-Shinojiya I found that the issue is the use of `--aot` in the `ng serve --aot` in the `npm start` command. Dunno this is happening, but now I can replicated it here !

**Comment by Arjun-Shinojiya** [comment]
@andrefarzat okay, anyways my issue solved with WindowModule. Thank you for your help.

**Comment by andrefarzat** [comment]
Finally I could understand. Upgrading the whole project to angular 8.x solved the issue. It was related to the difference in the `ViewChild` decorator. The fix will be in version 1.3.0 😸

**Comment by TravisLRiffle** [comment]
I'm on Angular 8 and having this issue as well.
ERROR TypeError: this.getPlotly(...).react is not a function
